### PR TITLE
Merkle Tree Updates

### PR DESCRIPTION
**Pull request by charlietilt**
## Summary
- updated the merkle tree lib to store all values as hex
- fixed a bunch of linting errors

### CODE REVIEW

File: src/lib/asteroid.js [hunk 1]
```diff
@@ -318,8 +318,8 @@ const BONUS_MAPS = [
 
 const PHI = Math.PI * (3 - Math.sqrt(5));
 const TWO_PI = 2 * Math.PI;
-const RESOURCE_OCTAVE_MUL = 5;
-const RESOURCE_OCTAVE_BASE = 3;
+// const RESOURCE_OCTAVE_MUL = 5;
+// const RESOURCE_OCTAVE_BASE = 3;
```
Comment (clexmond) [inline src/lib/asteroid.js:322]: Why are these commented out?

Comment (charlietilt) [reply src/lib/asteroid.js:322]: They are not currently used or exported

**Comment by clexmond** [comment]
Can you bump the versions (3 spots) to beta.26 in package.json and package-lock.json?

**Comment by charlietilt** [comment]
> Can you bump the versions (3 spots) to beta.26 in package.json and package-lock.json?

Done 🍻

**Review by clexmond — APPROVED** [review]

**Pull request merged**